lib/jsdom/living/helpers/stylesheets.js

```diff
@@ -2,7 +2,6 @@
 const cssom = require("cssom");
```

Comment:
This file is also very clean. It looks like the classes in Document-impl.js worked out really well.